src/chi/components/data-table/data-table.scss

```diff
@@ -0,0 +1,758 @@
+@import '_variables';
+@import '_mixins';
+
+$sizes: (
+  xs: (
+    padding: 0.25rem,
+  ),
+  sm: (
+    padding: 0.5rem,
+  ),
+  md: (
+    padding: 0.75rem,
+  ),
+  lg: (
+    padding: 1rem,
+  ),
+  xl: (
+    padding: 1.25rem,
+  )
+);
+
+.chi-data-table {
+  border-collapse: collapse;
+  display: block;
+  font-size: $font-size-base;
+  line-height: $line-height-sm;
+  overflow-x: auto;
+  overflow-y: hidden;
+  width: 100%;
+
+  @include respond-to(md) {
+    display: flex;
+    flex-direction: column;
+  }
+
+  label {
+    margin-bottom: 0;
+  }
+
+  @each $type in map-keys($sizes) {
+    &.-#{$type} {
+      .chi-data-table__head {
+        .chi-data-table__cell {
+          padding: map-get(map-get($sizes, $type), padding);
+        }
+      }
+
+      .chi-data-table__body {
+        .chi-data-table__cell {
+          padding: map-get(map-get($sizes, $type), padding);
```

Comment:
md is our default table cell size but on mobile it renders cells larger than the default padding.
With md applied: `padding: 0.75rem;`
Without md applied: `padding: 0.25rem 0.75rem;`

Shouldn't md on mobile render as`padding: 0.25rem 0.75rem;`?

Reply:
I don't know exactly how to solve this on the mobile view. We could add breakpoints for mobile and desktop for the different sizes but it would increase the size of chi.css and I think it is not the best scenario for this corner case use.